etl/firefox_legacy_etl.py

```diff
@@ -44,6 +44,7 @@ def _get_legacy_firefox_metric_summary(probe_data, activity_mapping):
             "type": probe["type"],
             "description": most_recent_metadata["description"],
             "bug_numbers": most_recent_metadata["bug_numbers"],
+            "details": most_recent_metadata["details"],
```

Comment:
This `details` field includes the bucket information, along with other data that might be useful. Here's a sample response, according to [Probe Info](https://mozilla.github.io/probe-scraper/#tag/telemetry/operation/getGeneral).

```{
  "value": {
    "histogram/A11Y_CONSUMERS": {
      "history": {
        "nightly": [
          {
            "cpp_guard": null,
            "description": "A list of known accessibility clients that inject into Firefox\nprocess space (see\nhttps://dxr.mozilla.org/mozilla-central/source/accessible/windows/msaa/Compatibility.h).\n",
            "details": {
              "high": 11,
              "keyed": false,
              "kind": "enumerated",
              "low": 1,
              "n_buckets": 12
            },
            "expiry_version": "never",
            "optout": true,
            "revisions": {
              "first": "320642944e42a889db13c6c55b404e32319d4de6",
              "last": "6f5fac320fcb6625603fa8a744ffa8523f8b3d71"
            },
            "versions": {
              "first": "56",
              "last": "59"
            }
          }
        ]
      },
      "name": "A11Y_CONSUMERS",
      "type": "histogram"
    }
  }
}

```